src/styles/block/nav.scss

```diff
@@ -24,7 +7,7 @@
 
   &__item {
     position: relative;
-    color: #060b35;
+    color: $main-color;
     list-style: none;
```

Comment:
you've already removed list style in reset.scss, no need to do it again

Reply:
done